Packs/Orca/Integrations/Orca/README.md

```diff
@@ -0,0 +1,80 @@
+Agentless, Workload-Deep, Context-Aware Security and Compliance for AWS, Azure, and GCP.
+This integration was integrated and tested with version xx of Orca
```

Comment:
```suggestion
This integration was integrated and tested with version ? of Orca
```
There is specific version?
if not, remove it.

Reply:
done